examples/article.amp.html

```diff
@@ -8,9 +8,14 @@
   <script async src="https://cdn.ampproject.org/v0.js"></script>
 
   <style amp-custom>
+    * {
+      outline: none;
```

Comment:
Removing outline is not good for accessibility. outline can be removed for elements that have a custom style for on focus but we should not do it as a blanket rule.